extensions/agroal/deployment/src/main/java/io/quarkus/agroal/deployment/devui/AgroalDevUIProcessor.java

```diff
@@ -37,8 +42,35 @@ void devUI(DataSourcesJdbcBuildTimeConfig config,
         }
     }
 
+    @BuildStep
```

Comment:
I'm wondering if we should not move this build step to its own processor (and enforce this as a good practice)

Reply:
Well, it's already in the dev ui processor - maybe it's too much.